### PR TITLE
Done

### PR DESCRIPTION
**Pull request by nscarlyon**

### CODE REVIEW

File: css/styles.css [hunk 1]
```diff
@@ -3714,6 +3715,24 @@ a:visited {
   transform: scale(1.1);
 }
 
+#natasha-carlyon-cover {
+  background-image: url(http://i.imgur.com/tlSo5x2.jpg)
```
Comment (bhollan) [inline css/styles.css]: Hi Natasha!  It's best practice to put URLs that are arguments into other functions inside `"`s.  Same thing for the other two.  After this is fixed, we can look at merging your changes in.

File: css/styles.css [hunk 2]
```diff
@@ -3714,6 +3715,24 @@ a:visited {
...
+}
+
+  #natasha-carlyon-card {
+    background-image: url(http://i.imgur.com/tQqGZtx.jpg)
```
Comment (bhollan) [inline css/styles.css]: `"`s around the URL please.  :)

File: css/styles.css [hunk 3]
```diff
@@ -3714,6 +3715,24 @@ a:visited {
...
+}
+
+#natasha-carlyon-card:hover {
+  background-image: url(http://i.imgur.com/tQqGZtx.jpg)
```
Comment (bhollan) [inline css/styles.css]: Third time's a charm! :)

**Comment by nscarlyon** [comment]
I added the new changes.=)

**Pull request merged**